packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -48,17 +51,13 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 		{
 			name: "vite-plugin-cloudflare",
 			config(userConfig, env) {
-				if (env.isPreview) {
-					return { appType: "custom" };
-				}
-
```

Comment:
One issue with removing this is that any logs/warnings emitted by the Wrangler `readConfig` function will be emitted for both the input and output Worker config files.